bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleStatusInfo.java

```diff
@@ -69,6 +74,7 @@ public RuleStatusInfo(RuleStatus status, RuleStatusDetail statusDetail, String d
         this.status = status;
         this.statusDetail = statusDetail;
         this.description = description;
+        this.properties = properties;
```

Comment:
how do these properties relate to the triggered state?

Reply:
No, imho that is not "natural" at all, it is rather the "easy" way... I really think that this is no information that is in any way related to the "RuleStatus". It is some information that you want to send out as an event for specific use cases. Couldn't the eventing be done by a special type of CEPTrigger? Its handler implementation could use the EventAdmin and post there whatever it likes to - without having any negative impact on the general infrastructure.

Reply:
@adimova Could you follow-up on this? Is it ok for you to move the specific information to the trigger implementation and revert the addition of properties to the RuleStatusInfo from this PR?

Reply:
Yes, that was my intention, to prepare new PR with "Add triggered state when a rule is triggered", 'cause this PR has merge conflicts, also we still should discuss CEP concept to get clean. So this PR can be closed.